### PR TITLE
Added Blog Post and Videos

### LINKED ISSUE #1783: Add blog post and videos in community guide
## My actions before raising this issue
- [ ] Followed the [troubleshooting guide](https://docs.openfaas.com/deployment/troubleshooting/)
- [ ] Read/searched [the docs](https://docs.openfaas.com/)
- [ ] Searched [past issues](/issues)

## Why do you need this?
Add following blog post and videos in community guide
https://midnightprogrammer.net/post/installing-openfaas-on-k3s-single-node/
https://www.youtube.com/watch?v=-8MrDWg6K6s

## Are you a GitHub Sponsor (Yes/No?)

Check at: https://github.com/sponsors/openfaas
- [ ] Yes
- [x] No

### PR DESCRIPTION
**Pull request by nitishkumar71**
## Description

Community guide has been updated for 
https://midnightprogrammer.net/post/installing-openfaas-on-k3s-single-node/
https://www.youtube.com/watch?v=-8MrDWg6K6s

## Motivation and Context

- [x] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))
- [ ] My issue has received approval from the maintainers or lead with the `design/approved` label
Closes #1783

## How Has This Been Tested?

Documentation change was done. So no test is required.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -71,6 +71,7 @@ It would be great to hear from you especially if you have any of the above and w
 
 | Title                                                              | Presenter       | Date      |
 | ------------------------------------------------------------------ | --------------- | --------- |
```
Comment (alexellis) [inline community.md:73]: I think I would put these into the blog posts section instead of the featured / official videos?

Comment (nitishkumar71) [reply community.md]: I have updated heading a little bit.

**Review by alexellis — APPROVED** [review]
LGTM thank you

**Comment by alexellis** [comment]
I found these by accident, do you think you can find more from late 2022 or early 2023 on Google?

I'm not certain of the syntax, but something like?

`openfaas after:2022-01-01`

**Pull request merged**